tests/lib/tools/setup_nested_hybrid_system.sh

```diff
@@ -140,10 +142,14 @@ run_muinstaller() {
     # run installation
     local install_disk
     install_disk=$(remote.exec "readlink -f /dev/disk/by-id/virtio-target")
-    remote.exec "sudo muinstaller \
-        -label ${label} \
-        -device ${install_disk} \
-        -rootfs-creator /snap/muinstaller/current/bin/mk-classic-rootfs.sh"
+    muinstaller_args="-label $label -device $install_disk -rootfs-creator /snap/muinstaller/current/bin/mk-classic-rootfs.sh"
+    if [ -n "$passphrase" ]; then
+        muinstaller_args="$muinstaller_args --passphrase $passphrase"
+    fi
+    if [ -n "$extra_muinstaller_args" ]; then
+        muinstaller_args="$muinstaller_args $extra_muinstaller_args"
+    fi
+    remote.exec "sudo muinstaller $muinstaller_args"
```

Comment:
I wonder if we could get rid of this big string pass as parameter to `remote.exec` and use lists instead.